WORKSPACE

```diff
@@ -21,78 +21,143 @@ workspace(name = "vaticle_typedb_driver")
 # Load @vaticle_dependencies #
 ##############################
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "io_bazel_rules_go",
+    integrity = "sha256-fHbWI2so/2laoozzX5XeMXqUcv0fsUrHl8m/aE8Js3w=",
+    urls = [
+        "https://mirror.bazel.BUILD/github.com/bazelbuild/rules_go/releases/download/v0.44.2/rules_go-v0.44.2.zip",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.44.2/rules_go-v0.44.2.zip",
+    ],
+)
+
+http_archive(
+    name = "bazel_gazelle",
+    integrity = "sha256-MpOL2hbmcABjA1R5Bj2dJMYO2o15/Uc5Vj9Q0zHLMgk=",
+    urls = [
+        "https://mirror.bazel.BUILD/github.com/bazelbuild/bazel-gazelle/releases/download/v0.35.0/bazel-gazelle-v0.35.0.tar.gz",
+        "https://github.com/bazelbuild/bazel-gazelle/releases/download/v0.35.0/bazel-gazelle-v0.35.0.tar.gz",
+    ],
+)
+
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
+
+load("//go:deps.bzl", "go_dependencies")
+# gazelle:repository_macro go/deps.bzl%go_dependencies
+go_dependencies()
+go_rules_dependencies()
+go_register_toolchains(version = "1.21.10")
+gazelle_dependencies()
+
+
 load("//dependencies/vaticle:repositories.bzl", "vaticle_dependencies")
+
 vaticle_dependencies()
 
 # Load //builder/bazel for RBE
 load("@vaticle_dependencies//builder/bazel:deps.bzl", "bazel_toolchain")
+
 bazel_toolchain()
 
 # Load //builder/python
 load("@vaticle_dependencies//builder/python:deps.bzl", python_deps = "deps")
+
 python_deps()
+
 load("@rules_python//python:repositories.bzl", "py_repositories", "python_register_toolchains")
+
 py_repositories()
+
 load("//python:python_versions.bzl", "register_all_toolchains")
+
 register_all_toolchains()
 
 # Load //builder/java
 load("@vaticle_dependencies//builder/java:deps.bzl", java_deps = "deps")
+
 java_deps()
 
 # Load //builder/kotlin
 load("@vaticle_dependencies//builder/kotlin:deps.bzl", kotlin_deps = "deps")
+
 kotlin_deps()
+
 load("@io_bazel_rules_kotlin//kotlin:repositories.bzl", "kotlin_repositories")
+
 kotlin_repositories()
+
 load("@io_bazel_rules_kotlin//kotlin:core.bzl", "kt_register_toolchains")
+
 kt_register_toolchains()
 
 # Load //builder/antlr
-load("@vaticle_dependencies//builder/antlr:deps.bzl", antlr_deps = "deps", "antlr_version")
+load("@vaticle_dependencies//builder/antlr:deps.bzl", "antlr_version", antlr_deps = "deps")
+
 antlr_deps()
 
 load("@rules_antlr//antlr:lang.bzl", "JAVA")
 load("@rules_antlr//antlr:repositories.bzl", "rules_antlr_dependencies")
+
 rules_antlr_dependencies(antlr_version, JAVA)
 
 # Load //builder/cpp
 load("@vaticle_dependencies//builder/cpp:deps.bzl", cpp_deps = "deps")
+
 cpp_deps()
 
 # Load //builder/csharp
 load("@vaticle_dependencies//builder/csharp:deps.bzl", dotnet_deps = "deps")
+
 dotnet_deps()
+
```

Comment:
These newlines look like a really aggressive auto formatting (and my auto formatting does not do it). I like the current state of the file, all the logical pieces are connected correctly, so let's restore the file and reapply only your `go` changes without this formatting. You can check the auto formatting settings in `IntelliJ` settings.

Reply:
It is still full of these newlines...